fusionauth/resource_fusionauth_tenant_test.go

```diff
@@ -242,11 +242,19 @@ func testTenantAccTestCheckFuncs(
 		resource.TestCheckResourceAttr(tfResourcePath, "password_validation_rules.0.validate_on_login", "true"),
 
 		// rate_limit_configuration
-		resource.TestCheckResourceAttrSet(tfResourcePath, "rate_limit_configuration"),
+		resource.TestCheckResourceAttrSet(tfResourcePath, "rate_limit_configuration.#"),
```

Comment:
Updated to avoid `Check failed: Check 90/106 error: fusionauth_tenant.test_ej4h1rwz6z: list or set attribute 'rate_limit_configuration' must be checked by element count key (rate_limit_configuration.#) or element value keys (e.g. rate_limit_configuration.0). Set element value checks should use TestCheckTypeSet functions instead.`